mne/beamformer/tests/test_external.py

```diff
@@ -89,6 +89,13 @@ def test_lcmv_fieldtrip(_get_bf_data, bf_type, weight_norm, pick_ori, pwr):
     else:
         stc_mne = apply_lcmv(evoked, filters)
 
+    if weight_norm == 'unit-noise-gain':
+        scale = np.sqrt(mne.make_ad_hoc_cov(evoked.info)['data'][0])
```

Comment:
I am not convinced by this. If a user does not specify a noise covariance matrix, do we really want to provide one that re-scales the output? Thoughts, @wmvanvliet @larsoner

Reply:
I think of it as: for unit-noise-gain, we have to scale the output to *something* based on what we want to assume the noise is (as it's not supplied, we have to make some choice here). We currently effectively scale it to the SI unit, e.g., T/m for gradiometers. This isn't a very realistic choice on our part (even if it's easy/simple/principled), since noise levels are never really this high in practice, and are generally a dozen orders of magnitude or so different.

Hence this PR can be thought of a proposal to change our assumption from assuming that we need to scale to unit noise under the assumption "1 T/m, 1 T, and 1 V for grad, mag, and EEG" to "5 fT/cm, 20 fT, and 0.2 µV for grad, mag, and EEG".

IIUC in this particular example, we generate noise that is very unrealistic in the sense that it is around a value of 1 in SI units. For realistic datasets (e.g., empty room, resting-state, tasks, etc.) the filters will instead give output values that are tiny. If a user decides to provide an actual noise cov, the scale of the output will change by 6-14 orders of magnitude or so (depending on their channel types) like in #11294. And if you use just mag in one filter vs just EEG in another in `main`, your output values will be different by six or seven orders of magnitude just because the SI units are different for these channel types (rather than at least being within an order of magnitude in this PR).

I'm still not 100% convinced by this line of reasoning, as both ways of thinking about it could make sense. But to me the practical considerations here push me to +0.5 on this change since it makes the default behave more nicely/consistently for users :shrug:

Reply:
AFAIK, unit-noise-gain assumes white noise, so ideally, the user should supply a noise covariance matrix. I see why not supplying anything is not ideal - but I am not so sure if we should put a band-aid on the problem by guesstimating some values that might or might not be true and signal to the user it's okay ... The plus-side of the approach we have is the consistent behavior across two major toolboxes.
FWIW, the LCMV option of "nai" is a unit-noise-gain beamformer that is scaled by the noise estimate of the signal.

Reply:
So maybe a `.. note::` or `.. warning::` for the `noise_cov` argument or in the `Notes` saying "if you don't supply a noise_cov, unit-noise-gain will assume noise is unity in the SI unit, which is unrealistic" or something like this?

Reply:
I think that is better than changing it.

Reply:
> And if you use just mag in one filter vs just EEG in another in main, your output values will be different by six or seven orders of magnitude just because the SI units are different for these channel types

Don't we have an error for that? You can only mix channel types if you have a noise cov. I'd rather that the user provides one explicitly than implicitly start scaling the output.

Reply:
> Don't we have an error for that? You can only mix channel types if you have a noise cov.

Yes we have an error message if you try to use them simultaneously. I'm talking about a case where you make one set of filters for mag, then a different one for EEG. The scales will be drastically different just because the SI units of these channel types differ.